Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM archlinux:base-devel AS build
+
+RUN pacman -Syu --noconfirm meson git emscripten jre-openjdk-headless unzip npm
+
+# Uncomment if you want emsdk
+# RUN useradd -m --shell=/bin/false build && \
+#     usermod -L build && \
+#     echo "build ALL= NOPASSWD: /usr/bin/pacman" > /etc/sudoers.d/build
+#
+# USER build
+#
+# WORKDIR /home/build
+#
+# RUN git clone https://aur.archlinux.org/emsdk.git && \
+#     cd emsdk && makepkg -sfi --noconfirm
+#
+# USER root
+
+WORKDIR /build
+
+COPY . .
+
+RUN rm -rf build-emscripten-wasm32 github-pages && \
+    source /etc/profile.d/emscripten.sh && \
+    bash scripts/gh-pages-deploy.sh
+
+FROM node:current-alpine
+
```

Comment:
The nginx docker seems pretty easy to configure. Just copy the output to /usr/share/nginx/html and you're all set.

Reply:
I'll test it out.